### DISCUSSION TITLE
csc

### DISCUSSION THREAD

**Discussion by GED228** [community/community#84119] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Palindrome
Write a function that checks if a character string passed as a parameter is a palindrome (reads the same way from left to right and from right to left).
Carry out tests of your function on the following palindrome sentences:
•
To the Emir, Asimov vomited his rhyme, there
•
At the stage, amaze her!
•
This landmark, Perec.
•
She brings Maëlle
Bonus: Write a second function that generates palindromes

**Comment by queenofcorgis** [comment]
Hi @GED228 ,
The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was flagged and removed because it violated our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. Asking folks for help with homework or test questions may discourage engagement and responses.

If this content is posted again, your Community account will be subject to immediate and indefinite suspension.
For additional guidance, please review our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service).

Thank you!
